Source/Krypton Components/Directory.Build.targets

```diff
@@ -139,33 +139,33 @@
 				<PackageId Condition="'$(TFMs)' == 'all'">$(PackageId).Canary</PackageId>
 
 
-				<Description Condition="'$(TFMs)' == 'all'">An update to Component factory's krypton toolkit to support .NET Framework 4.6.2 - 4.8.1 and .NET 6 - 8. $(Description) This package supports all .NET Framework versions starting .NET Framework 4.6.2 - 4.8.1 and .NET 6 - 7. Also, all libraries are included targeting each specific framework version for performance purposes. To view all of the standard toolkit package latest version information, please visit: https://github.com/Krypton-Suite/Krypton-Toolkit-Suite-Version-Dashboard/blob/main/Documents/Modules/Standard/Krypton-Toolkit-Suite-Standard-Modules.md To find out what's new, please visit: https://github.com/Krypton-Suite/Standard-Toolkit/blob/canary/Documents/Help/Changelog.md This package is for those who want to try out the latest features before deployment.</Description>
+				<Description Condition="'$(TFMs)' == 'all'">An update to Component factory's krypton toolkit to support .NET Framework 4.6.2 - 4.8.1 and .NET 6 - 9. $(Description) This package supports all .NET Framework versions starting .NET Framework 4.6.2 - 4.8.1 and .NET 6 - 7. Also, all libraries are included targeting each specific framework version for performance purposes. To view all of the standard toolkit package latest version information, please visit: https://github.com/Krypton-Suite/Krypton-Toolkit-Suite-Version-Dashboard/blob/main/Documents/Modules/Standard/Krypton-Toolkit-Suite-Standard-Modules.md To find out what's new, please visit: https://github.com/Krypton-Suite/Standard-Toolkit/blob/canary/Documents/Help/Changelog.md This package is for those who want to try out the latest features before deployment.</Description>
```

Comment:
Please read through as it still has `.Net 6 - 7` in the text..

Reply:
Done